src/track/cue.h

```diff
@@ -28,8 +28,10 @@ class Cue : public QObject {
         BEAT    = 3,
         LOOP    = 4,
         JUMP    = 5,
-        INTRO   = 6,
-        OUTRO   = 7,
+        INTRO_START = 6,
+        INTRO_END   = 7,
```

Comment:
I was thinking this would be implemented by reusing Cue's ability to mark ranges which is already used for the LOOP cue type instead of having separate types for the start and end of each range. What do you think?

Reply:
I tried that approach, but there is a problem since AnalyzerSilence needs to place just outro end cue point without outro start cue point, which means that we need range without starting point. I was thinking of solving this by having start set to -1 and making length store the position of end marker for this case, but this seems like an ugly hack to me, so I decided for a simpler approach.

Reply:
I was thinking of having AnalyzerSilence place the outro start and end at the same point. It would then be the user's choice to move the start point if they want.